### PR TITLE
neverball, bump, fix references to libpng16, libSDL2_*

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: games-puzzle/neverball/neverball-1.6.0~20190609.recipe [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+COPYRIGHT="2002-2019 The Neverball Team"
 LICENSE="GNU GPL v2"
-REVISION="5"
-SOURCE_URI="https://neverball.org/neverball-$portVersion.tar.gz"
-CHECKSUM_SHA256="73fe63cca4f96e2d355480d03bc0b2904e83a0abdf65fe8c52db5cc3cca88fa0"
+REVISION="1"
+srcGitRev="cd7dfc9464e242f100dabe711863f3a943cec1d5"
+SOURCE_URI="https://github.com/Neverball/neverball/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="ee1b9ffb4c32d0e1c0993f5900c94bbc35d5b9e836e487c7a4aa71c547c1e87a"
+SOURCE_FILENAME="neverball-$srcGitRev-$portVersion.tar.gz"
```
Comment (Begasus) [inline games-puzzle/neverball/neverball-1.6.0~20190609.recipe]: This probably needs to be reverted, first $portVersion then $srcGitRev?

**Comment by Begasus** [comment]
Created a PR to upstream the patchset, maybe wait for that to be merged? https://github.com/Neverball/neverball/pull/191

**Comment by Begasus** [comment]
Not tested on x86_64, so would be nice if someone could step in to check that :)

**Comment by Begasus** [comment]
Related to https://github.com/haikuports/haikuports/issues/3878

File: games-puzzle/neverball/neverball-1.6.0~20190609.recipe (hunk 1 above)
Comment (korli) [reply games-puzzle/neverball/neverball-1.6.0~20190609.recipe]: yes

**Comment by korli** [comment]
neverball-1.6.0~20190609.recipe should be named neverball-1.6.1~20190609.recipe as it's already the a prerelease of the next version. The package manager won't upgrade from 1.6.0 to 1.6.0~20190609.

**Comment by korli** [comment]
tested on x86_64, applied in 8f812ccc02ac9edb31c4217203278de59f478dc9

**Pull request closed without merging**